src/backend/agent.js

```diff
@@ -442,9 +450,20 @@ export default class Agent extends EventEmitter {
     event.stopPropagation();
 
     const target = ((event.target: any): HTMLElement);
-
     // Don't pass the name explicitly.
     // It will be inferred from DOM tag and Fiber owner.
     showOverlay(target);
+
+    const id = this.getIDForNode(target);
+    if (id !== null) {
```

Comment:
Is the null check correct? If we aren't currently moused over an element that matches a fiber, do we still want to select the most recent one?

Reply:
Do you think we should clear the selection completely?

Reply:
I don't think so. In this case, we should maybe just leave the most recently selected thing as-is?